modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -481,7 +496,125 @@ public void testAddObjectAction() throws Exception {
 
 			_assertWebhookObjectAction(
 				"Peter", ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE,
-				"Peter", WorkflowConstants.STATUS_APPROVED);
+				_objectDefinition, "Peter", WorkflowConstants.STATUS_APPROVED);
+
+			// Create root context
+
+			ObjectDefinition objectDefinitionA =
+				ObjectDefinitionTestUtil.addObjectDefinition(
+					false, _objectDefinitionLocalService,
+					Collections.singletonList(
+						ObjectFieldUtil.createObjectField(
+							ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+							ObjectFieldConstants.DB_TYPE_STRING, "First Name",
+							"firstName")));
+
+			ObjectDefinition objectDefinitionAA =
+				ObjectDefinitionTestUtil.addObjectDefinition(
+					false, _objectDefinitionLocalService,
+					Collections.singletonList(
+						ObjectFieldUtil.createObjectField(
+							ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+							ObjectFieldConstants.DB_TYPE_STRING, "First Name",
+							"firstName")));
+
+			ObjectRelationship objectRelationshipA_AA =
+				ObjectRelationshipTestUtil.addObjectRelationship(
+					_objectRelationshipLocalService, objectDefinitionA,
+					objectDefinitionAA);
+
+			ObjectDefinition objectDefinitionAAA =
+				ObjectDefinitionTestUtil.addObjectDefinition(
+					false, _objectDefinitionLocalService,
+					Collections.singletonList(
+						ObjectFieldUtil.createObjectField(
+							ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+							ObjectFieldConstants.DB_TYPE_STRING, "First Name",
+							"firstName")));
```

Comment:
That is right. But, in order to be able to delete the `objectDefinition`'s that were created, we'll need to add a field to them. Since they need to have at least one custom field